mfts/papermark
# Accessibility Advocate- Added close button in signup page

### Changes Implemented:
- Added a prominent close button ("X") to the signup page.
- Clicking the close button now dismisses the signup/login page and redirects the user to the landing/home page.

### Verification Steps:
- Navigate to the signup/login page.
- Click the close button ("X").
- Verify that the signup page is dismissed and the user is redirected to the landing/home page.
 
### Result:
The close button functionality has been successfully implemented, enhancing user experience and accessibility.

### Status:
Completed and ready for review.
![Screenshot 2024-10-11 120002](https://github.com/user-attachments/assets/6d7dee0f-a45d-4242-bb6c-03c1073a114f)
